app/javascript/controllers/toggle_blank_controller.js

```diff
@@ -0,0 +1,21 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["toggle"]
+
+  toggleTargetConnected(el) {
```

Comment:
A light controller to clear the "Number of Episodes" / "End Date" fields, when you enter a non-blank value in the other one.